libsplinter/src/rest_api/auth/rbac/rest_api/actix_web_1/roles.rs

```diff
@@ -183,6 +197,34 @@ fn add_role(
     )
 }
 
+fn get_role(
+    req: HttpRequest,
+    role_based_auth_store: web::Data<Box<dyn RoleBasedAuthorizationStore>>,
+) -> Box<dyn Future<Item = HttpResponse, Error = Error>> {
+    let role_id = req.match_info().get("role_id").unwrap_or("").to_string();
+    Box::new(
+        web::block(move || {
+            role_based_auth_store
+                .get_role(&role_id)
+                .map_err(SendableRoleBasedAuthorizationStoreError::from)
+        })
+        .then(|role_res| {
+            Ok(match role_res {
+                Ok(Some(role)) => HttpResponse::Ok().json(json!({
+                    "data": RoleResponse::from(&role),
```

Comment:
Why put the role in a `data` field? Do we do this elsewhere?

Reply:
We do that in the node registry api, biome register, key management post response, and get response, etc. In fact, it seems like the endpoints that don't do that are the exception, not the rule.